src/main/java/com/indeed/jiraactions/JiraActionsIndexBuilderConfig.java

```diff
@@ -20,5 +20,8 @@
     String getEndDate();
     int getJiraBatchSize();
     String getIndexName();
+    boolean buildJiraIssues();
+    int getJiraIssuesRange();
```

Comment:
What does this represent? Looking at the code, it appears to be the number of months back we will look for issues from?